src/Search/Search.test.tsx

```diff
@@ -1,3 +1,23 @@
 import React from "react";
-import { render, screen } from "@testing-library/react";
-import Search from './Search'
\ No newline at end of file
+import { render, screen, getByPlaceholderText, fireEvent } from "@testing-library/react";
+import Search from './Search'
+
+describe('Search', () => {
+  beforeEach(() => {
+    render(<Search />)
+  })
+
+  it('should render a search bar', () => {
+    const search = screen.getByPlaceholderText('Search city, zip, or county')
+    expect(search).toBeInTheDocument()
+  })
+
+  it("should reflect change in value when data is input in form", () => {
```

Comment:
Thanks for getting these tests done to ensure the search bar is working properly!